scripts/infrastructure/giskard-docker-start.service

```diff
@@ -10,14 +10,13 @@ WorkingDirectory=/home/ec2-user
 Type=oneshot
 RemainAfterExit=yes
 
-# change this to branch main before merging
-ExecStartPre=-/bin/bash -c "/bin/curl -f 'https://raw.githubusercontent.com/Giskard-AI/giskard/feat/ami/docker-compose.ami.yml' -o docker-compose.ami.yml"
+ExecStartPre=-/bin/bash -c "/bin/curl -f 'https://raw.githubusercontent.com/Giskard-AI/giskard/main/docker-compose.ami.yml' -o docker-compose.ami.yml"
 ExecStartPre=-/bin/bash -c "/bin/curl -f 'https://raw.githubusercontent.com/Giskard-AI/giskard/main/docker-compose.yml' -o docker-compose.yml"
 ExecStartPre=-/usr/local/bin/docker-compose pull --quiet
 ExecStart=/usr/local/bin/docker-compose -f docker-compose.yml -f docker-compose.ami.yml up -d
 
 ExecStartPost=/bin/bash -c "echo 'Pruning docker images...'"
-ExecStartPost=-docker image prune -f
+ExecStartPost=-/usr/local/bin/docker image prune -f
```

Comment:
Hello @andreybavt je viens de checker et c'est pas correct, le path to docker c'est /usr/bin par default (et le path to docker-compose est different car il est installe differement, pas par yum)
Si tu es ok je vais pusher ca directement sur main?

Reply:
Salut @reneab , oui, push le directement, merci!

Reply:
ok et je rajoute un petit script aussi d'installation de docker et docker-compose sur AWS Linux, ca manquait, sorry

Reply:
bon en fait j'ai cree une petit pull request c'est mieux 
https://github.com/Giskard-AI/giskard/pull/184